Assets/Scripts/Battlefield/TurnBrain.cs

```diff
@@ -108,7 +108,10 @@ void movementState()
             if (Physics.Raycast(ray, out hit, 100, tileMapLayerMask))
             {
                 Tile currentTile = hit.collider.GetComponent<Tile>();
-                tileIndictor.transform.position = currentTile.GetPos();
+                if (Time.timeScale >0)
+                {
+                    tileIndictor.transform.position = currentTile.GetPos();
+                }
```

Comment:
Can we make this more robust by having a Pause flag in some relevant Manager or something because this is much too vague and could impede us later?